.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,43 @@
-Thank you for contributing to PlasmaPy!  Please check out our development guide at:
+<!--
+Thank you for contributing to PlasmaPy! Here's a bunch of pointers to
+make things easier for all of us:
 
-http://docs.plasmapy.org/en/master/development/index.html
+* If this PR will solve an issue tracked by GitHub, please add a phrase like
+  "Closes #404." to automatically close the issue once the pull request is
+  merged .  If your PR will not completely solve the issue, please still
+  reference the issue. For an overview of this functionality, see
+  https://help.github.com/articles/closing-issues-using-keywords/
 
-Here are a few things to keep in mind when making a pull request (PR).
+* Remember to add some description of your changes in this text box.
 
 * Tests and docstrings are required for new or changed functionality.
-* Please take care to write helpful commit messages: https://chris.beams.io/posts/git-commit/
-* As a PR nears completion, fellow developers will likely suggest changes
-  to be made before the PR is merged.
-* Every pull request should include a changelog entry. Please see
-  `changelog/README.rst` for instructions.
-* PlasmaPy follows the PEP 8 style guide: https://www.python.org/dev/peps/pep-0008
-* Please double check that a new PR will not duplicate an existing PR.
-* If this PR will solve an issue tracked by GitHub, please add a phrase like
-  "Closes #404." to automatically close the issue once the pull request is
-  merged.  If your PR will not completely solve the issue, please still
-  reference the issue.
+  Please make sure tests are passing before requesting a review - they
+  will pop up at the bottom, in the Checks box. If you're unsure why
+  they're failing, ask!
+  
+* If your pull request is not yet ready for review, add a `[WIP]`
+  label to its name and remove it once it's ready.
```

Comment:
We haven't been doing this step lately, so I'd be in favor of removing it so that there are fewer steps.  Plus, there now seems to be a "draft" feature for PRs in GitHub, though that's another thing I still need to learn how to do.

Reply:
Sure, let's cut it!